package.json

```diff
@@ -4,25 +4,61 @@
   "description": "Pocketdex is an indexer for the Shannon implementation of the pocket network protocol.",
   "main": "dist/index.js",
   "scripts": {
+    "preinstall": "npx only-allow yarn",
+    "postinstall": "yarn run env:prepare",
+
     "codegen": "yarn exec subql codegen",
     "build": "./scripts/build.sh",
-    "prepack": "rm -rf dist && yarn run build",
+    "watch:build": "./scripts/build.sh no-lint",
+    "lint": "yarn exec eslint src --ext .ts",
+    "lint:fix": "yarn exec eslint src --ext .ts --fix",
+    "format": "yarn exec prettier --loglevel error --write 'src/*.{js,ts,jsx,tsx}'",
+    "format:ci": "yarn exec pretty-quick --verbose --check --pattern 'src/*.{js,ts,jsx,tsx}' --branch origin/main",
     "test": "yarn run build && subql-node-cosmos test",
-    "vendor:setup": "echo 'installing and building vendor modules' && yarn vendor:cosmjs:install && yarn vendor:cosmjs:build && yarn vendor:subql:install && yarn install && yarn vendor:subql:build",
-    "vendor:cosmjs:install": "cd vendor && yarn workspace cosmjs-monorepo-root install",
-    "vendor:subql:install": "cd vendor && yarn workspace @subql/node-cosmos install",
-    "vendor:cosmjs:build": "cd vendor && yarn workspace cosmjs-monorepo-root run build",
-    "vendor:subql:build": "cd vendor && yarn workspace @subql/node-cosmos run build",
-    "vendor:clean": "echo 'cleaning vendor modules' && cd vendor && find . -name node_modules | xargs rm -rf && find . -name dist | xargs rm -rf",
-    "docker:build": "env sh ./scripts/dotenv-check.sh && docker compose build",
-    "docker:build:no-cache": "env sh ./scripts/dotenv-check.sh && docker compose build --no-cache",
-    "docker:start": "docker compose pull && docker compose  up --remove-orphans",
-    "docker:stop": "docker compose stop",
-    "docker:clean": "docker compose down -v",
-    "docker:tunnel": "./scripts/proxy-tunnel.sh"
+    "test:ci": "env node vendor/subql-cosmos/packages/node/bin/run test",
+
```

Comment:
I'm noticing that when I run `yarn install`, all blank lines are removed from the package.json. Let's remove them preemptively so that this doesn't confuse other contributors.
```suggestion
```